style.css

```diff
@@ -0,0 +1,69 @@
+    header {
+        position: fixed;
+        background-color: #ffffff;
+        width: 100%;
+        height: 70px;
+    }
+    
+    body {
+        background-color: black;
+        margin: 0px;
+    }
+    
+    .logo {
+        width: 70px;
+        height: 11px;
+        margin-top: 27px;
```

Comment:
You can't use margins to align your elements as you did. It's not responsive. Google this "css flexbox" to understand how to do it right and fix **ALL** you margins!
![image](https://user-images.githubusercontent.com/45945732/82830741-35656100-9ebf-11ea-9d47-00572367d9e0.png)

Reply:
that's fixed